configure.ac

```diff
@@ -134,6 +134,15 @@ if test "$enable_baytrail" = "yes"; then
 AC_DEFINE([__PLATFORM_BYT__], [1], [Defined to 1 if --enable-baytrail="yes" ])
 fi
 
+AC_ARG_ENABLE(openssl,
```

Comment:
better change logical like this.
1. if --eanble-md5=no, disable -2 mode
2. automatic check openssl, if it's exist, disable the -2 mode